### PR TITLE
ADEN-8641 Remove JS, CSS, spec, HTML, resource, image and dist files

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: extensions/wikia/AdEngine3/templates/AdEngine3_AdEmptyContainer.php [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
-    $slotId = AdEngine3::isEnabled() ? strtolower( $slotName ) : $slotName;
-?>
+<?php $slotId = strtolower( $slotName ); ?>
 
 <?php if ($showAd): ?>
 	<!-- BEGIN CONTAINER: <?= htmlspecialchars( $slotId ) ?> -->
 	<div id="<?= htmlspecialchars( $slotId ) ?>">
 		<script>
-			window.adslots2.push(<?= json_encode([$slotId]) ?>);
+            window.adslots2.push(<?= json_encode([$slotId]) ?>);
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine3/templates/AdEngine3_AdEmptyContainer.php]: Indent :p

File: extensions/wikia/AdEngine3/templates/AdEngine3_Ad.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	<?php endif; ?>
+	<!-- BEGIN SLOTNAME: <?= htmlspecialchars( $slotId ) ?> -->
+	<div id="<?= htmlspecialchars( $slotId ) ?>" class="wikia-ad noprint default-height">
+		<? if ($includeLabel): ?>
+			<label class="wikia-ad-label"><?= wfMessage( 'adengine-advertisement' )->escaped() ?></label>
+		<? endif; ?>
+		<? if ($addToAdQueue): ?>
+			<script>
+				<? if ($onLoad) { ?>
+                wgAfterContentAndJS.push(function () {
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine3/templates/AdEngine3_Ad.php]: Indents :p

File: extensions/wikia/AdEngine3/AdEngine3ApiController.class.php [hunk 3]
```diff
@@ -4,6 +4,11 @@ class AdEngine3ApiController extends WikiaController
 {
 	const DEFAULT_TEMPLATE_ENGINE = WikiaResponse::TEMPLATE_ENGINE_MUSTACHE;
 
+	public function getCollapse()
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine3/AdEngine3ApiController.class.php]: Seems not needed

File: extensions/wikia/AdEngine3/templates/AdEngine3_Ad.php [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<?php $slotId = strtolower( $slotName ); ?>
+
+<?php if ($showAd): ?>
+	<?php // For Ad Mix experiment - XW-3156 ?>
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine3/templates/AdEngine3_Ad.php]: Comment seems not needed anymore

File: extensions/wikia/AssetsManager/config.php [hunk 5]
```diff
@@ -34,92 +34,11 @@
...
-		'//resources/wikia/modules/viewportObserver.js',
-		'//extensions/wikia/AdEngine/js/run/desktop.run.js'
-	],
-];
-
 $config['recirculation_js'] = [
 	'type' => AssetsManager::TYPE_JS,
 	'skin' => [ 'oasis' ],
 	'assets' => [
-        	'//resources/wikia/modules/eventLogger.js',
```
Comment (fraszczakszymon) [inline extensions/wikia/AssetsManager/config.php:122]: 💎 😄

File: extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php [hunk 6]
```diff
@@ -82,9 +82,6 @@ class WikiaRobots {
 		'/fandom?p=',
 		//Mobile Wiki search URL
 		'/search',
-
-		// AdEngine recovery api
-		'/wikia.php?controller=AdEngine2ApiController&method=getBTCode'
```
Comment (fraszczakszymon) [inline extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php:87]: Should we add here the other rec endpoints from AE3?

File: extensions/wikia/AdEngine3/templates/AdEngine3_AdEmptyContainer.php (hunk 1 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine3/templates/AdEngine3_AdEmptyContainer.php]: Fixed!

File: extensions/wikia/AdEngine3/templates/AdEngine3_Ad.php (hunk 2 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine3/templates/AdEngine3_Ad.php]: Fixed too!

File: extensions/wikia/AdEngine3/AdEngine3ApiController.class.php (hunk 3 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine3/AdEngine3ApiController.class.php]: Yep, removed.

File: extensions/wikia/AdEngine3/templates/AdEngine3_Ad.php (hunk 4 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine3/templates/AdEngine3_Ad.php]: Agree, removed.

File: extensions/wikia/AssetsManager/config.php (hunk 5 above)
Comment (dmnsobczak) [reply extensions/wikia/AssetsManager/config.php:122]: Ordnung muss sein 🇩🇪

File: extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php (hunk 6 above)
Comment (dmnsobczak) [reply extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php:87]: Yep, added

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**